### DISCUSSION TITLE
Local Development With TTS and Trainer Instructions

### DISCUSSION THREAD

**Discussion by iprovalo** [coqui-ai/TTS#1637] [General Q&A] [answered]
Are there instructions on how to set the local dev environment for developing with both the [TTS](https://github.com/coqui-ai/TTS) and the [Trainer](https://github.com/coqui-ai/Trainer)?

**Comment by erogol** [comment, marked as answer]
https://github.com/coqui-ai/TTS/blob/dev/CONTRIBUTING.md

**Comment by iprovalo** [comment]
@erogol thank you!
